### DISCUSSION TITLE
NOOB Q: is it possible to attach CSS like the app?

### DISCUSSION THREAD

**Discussion by sinkr** [epwalsh/obsidian.nvim#571] [Q&A] [answered]
Love the plugin!  Sorry if this is so obvious, but is it possible to attach CSS to notes like [this one](https://github.com/obsidian-tasks-group/obsidian-tasks) to get extra icons, etc?

**Comment by epwalsh** [comment, marked as answer]
That's outside the scope of this plugin. You could probably achieve something similar uses Neovim extension marks, but it's probably not worth the effort (working with extmarks is pretty complex).

**Reply by sinkr** [reply]
Thanks for the response!  This keeps me from digging deeper into something I don't fully understand.
